iris/static/js/lecturer_session.js

```diff
@@ -1,38 +1,42 @@
 const button_start = document.getElementById("button_start");
 const button_stop = document.getElementById("button_stop");
-const feedbackmsg = document.getElementById("lecturer_message")
+const feedbackmsg = document.getElementById("lecturer_message");
 let socket = io();
 socket.emit('join', {'course_id': courseID});
 
-if(sessionActive == "True"){
+function disableStart(){
     button_start.disabled = true;
     button_stop.disabled = false;
-}else{
+    feedbackmsg.innerHTML = "Session active";
+}
+
+function disableStop(){
```

Comment:
Not sure. It needs to know what button it is going to be disabled depending on if there is an active session or not. 
It could be made into a more general function, but then the if statement needs to include the code that is currently in the disabling functions. And you would have to check which button is disabled to display the correct text (if we are going to have some kind of feedback). All in all I guess it would not be less code.